src/spatial/collision.js

```diff
@@ -332,16 +325,16 @@ Crafty.c("Collision", {
      * @see .checkHits
      * @see .hit
      */
-    onHit: function (comp, callback, callbackOff) {
+    onHit: function (component, hit, noHit) {
```

Comment:
I actually liked the previous parameter names better, as the `-Off` part reflected their purpose better - function to be called after collision resolved.
Some suggestions: `callbackOn/-Off`, `hitOn/-Off`, etc.